payments-ui-core/src/main/java/com/stripe/android/ui/core/PaymentsTheme.kt

```diff
@@ -43,6 +47,12 @@ object PaymentsThemeConfig {
         return if (isDark) colorsDark else colorsLight
     }
 
+    @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+    object ShapeConfig {
```

Comment:
Can we have a different name other than `Config`? Maybe something like `ShapeProperties`, `ShapeDimensions`, or something like that. These would typically belong in a dimens.xml file, so I thought this would be appropriate. Otherwise, if this is a common pattern we see else where, I am also ok with this.

Reply:
I think we could split these out into two classes: `PaymentsThemeColors` and `PaymentsThemeShapes`, what do you think?

Reply:
We're actively forming the pattern now haha. So I appreciate the naming help. I just pushed a change renaming this config to just `Shapes`. I think it makes more sense. Thoughts?

For now I want these to be nested. This object is going to look very similar to the public API.